.github/workflows/elixir.yml

```diff
@@ -2,25 +2,58 @@ name: Elixir CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [master]
   pull_request:
-    branches: [ master ]
+    branches: [master]
 
 jobs:
   build:
     runs-on: ubuntu-latest
+
     strategy:
       matrix:
-        otp: ['21.0', '22.0']
-        elixir: ['1.7.2', '1.10.1']
+        elixir: ['1.13', '1.12', '1.11', '1.10']
+        otp: ['24.x', '23.x', '22.x']
+        include:
+          - lint: lint
+            elixir: '1.13'
+            otp: '24.x'
```

Comment:
ℹ️  Tells the action that it should only set the `lint` attribute when the Elixir version is `"1.13"` and the OTP version is: `"24.x"`. This prevents the Credo and the linting step to run on **every** matrix version.